test/fixtures/previous.comment.html

```diff
@@ -8,6 +8,8 @@ <h1>HTTP</h1>
 <p>Can start after asterisk *<a href="https://a.b">https://a.b</a>.</p>
 <p>Can start after underscore *_<a href="https://a.b">https://a.b</a>.</p>
 <p>Can start after tilde ~<a href="https://a.b">https://a.b</a>.</p>
+<p>Cannot start after greater than >http://www.example.com</p>
+<p>Cannot start after equals sign =http://www.example.com</p>
```

Comment:
This file is generated from GitHub (`UPDATE=1 npm test`), I’m assuming you added these manually, and that the actual GH results are different?